ivy_lint/formatters/function_ordering.py

```diff
@@ -0,0 +1,134 @@
+import re
+import ast
+from typing import Tuple, List
+
+from ivy_lint.formatters import BaseFormatter
+
+HEADER_PATTERN = re.compile(r"#\s?(-{0,3})\s?(Helpers|Main|API Functions)\s?(-{0,3})\s?#\n#\s?(-{7,15})\s?#\n(?:\s*\n)*")
```

Comment:
I think this regex is a bit complicated? Why do you use specific number of dashes?

Reply:
This is soo that we delete and rewrite headers for segregation every time. This regex took a lot of trial and error but it's working perfectly now